tests/collecting/CollectorsTest.py

```diff
@@ -396,3 +396,12 @@ def test_get_all_bears_names(self):
             self.assertSetEqual(
                 set(names),
                 set(TEST_BEAR_NAMES))
+
+    def test_bears_sorting(self):
+        with bear_test_module():
+            names = get_all_bears()
```

Comment:
it isnt populated with "names"; they a Bear objects?

Anyway, you can get coverage by putting it to use at `test_get_filtered_bears`.

It is too basic a wrapper around `sorted` to need its own tests